tests/Backend/Mixed/SharingTest.php

```diff
@@ -1409,20 +1416,50 @@ function (array $column) {
 
     public function invalidSharingTypeData()
     {
-        return [
-            'non existing type' => [
-                'global',
-            ],
-            'sharing to specifix projects' => [
-                'specific-projects',
-            ],
-            'sharing to specifix users' => [
-                'specific-users',
-            ],
+        yield 'non existing type sync' => [
+            'global',
+            false,
+        ];
```

Comment:
Co když budeme chtít přidat další věc k otestování? Budeme muset myslet na to, abysme přidali sync i asynch verzi. Bylo by IMHO lepší udělat `foreach ([true, false] as $async)` a v něm yieldovat